CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ option(OPT_BUILD_SDRPLAY_SOURCE "Build SDRplay Source Module (Depedencies: libsd
 option(OPT_BUILD_PLUTOSDR_SOURCE "Build PlutoSDR Source Module (Depedencies: libiio, libad9361)" ON)
 option(OPT_BUILD_HACKRF_SOURCE "Build HackRF Source Module (Depedencies: libhackrf)" ON)
 option(OPT_BUILD_RTL_SDR_SOURCE "Build RTL-SDR Source Module (Depedencies: librtlsdr)" ON)
+option(OPT_BUILD_RX888_SOURCE "Build RX-888 Source Module (Depedencies: libsddc)" ON)
```

Comment:
Wrong dependency listed, should be `libusb-1.0`

Reply:
Thanks for advice

It's anyway far from master, I will try to find a time to rebase

Reply:
With libusb but without libsddc it self this will not compile, maybe switch off by default and wait until libsddc will be more finished?